src/FakeItEasy/FakeItEasy.csproj

```diff
@@ -280,7 +280,10 @@
   <ItemGroup />
   <ItemGroup>
     <EmbeddedResource Include="Creation\CastleDynamicProxy\DynamicProxyResources.resx" />
-    <EmbeddedResource Include="ExceptionMessages.resx" />
+    <EmbeddedResource Include="ExceptionMessages.resx">
+      <Generator>PublicResXFileCodeGenerator</Generator>
+      <LastGenOutput>ExceptionMessages1.Designer.cs</LastGenOutput>
+    </EmbeddedResource>

```

Comment:
Yeah I was struggling with this change. The .Designer.cs file is actually unlinked from the .resx in version control, so VS kept trying to rebuild it, and finding a duplicate it would create ExceptionMessages1.Designer.cs which I then had to delete once I copied the piece manually of what I wanted. Not sure what the deal is here. I'll clear up some of these things soon. Do you want me to fix the auto-building of this file?